### PR TITLE
Update docs for windows container resources

### PR DESCRIPTION
**Pull request by feiskyer**
Update docs for windows container resources and Hyper-V containers.

Feature: https://github.com/kubernetes/features/issues/547

### CODE REVIEW

**Comment by feiskyer** [comment]
@Bradamant3 Could you help to approve this PR?

**Comment by xiangpengzhao** [comment]
@feiskyer if you preview this doc, the format is broken in some sections. Since you're here, could you please fix it?

The broken section is from this sentence `More detailed build instructions will be maintained ...`

ADD: another nit:
The example around here has bad indent.
`Windows pod with secrets mapped to environment variables`

**Comment by Bradamant3** [comment]
@feiskyer I fixed some more conflicts that I introduced because of my merge of master to the 1.10 branch. But it looks to me as though some content might have gotten dropped that should be retained -- notably, the `Next steps and resources` section at the bottom of the file. Could you please double-check? Thanks!

**Comment by feiskyer** [comment]
@Bradamant3 Thanks, but there are some problems of merge. I pushed a new correct commit, PTAL

**Comment by Bradamant3** [comment]
Thanks @feiskyer!

/approve

File: docs/getting-started-guides/windows/index.md [hunk 1]
```diff
@@ -434,14 +434,65 @@ spec:
...
+          limits:
+            memory: "128Mi"
+            cpu: 2
+        ports:
+        - containerPort: 80
+```
+
+### Hyper-V Containers
+
+Hyper-V containers are supported as experimental in v1.10. To create a Hyper-V container, kubelet should be started with feature gates `HyperVContainer=true` and Pod should include annotation `experimental.windows.kubernetes.io/isolation-type=hyperv`.
```
Comment (michmike) [inline docs/getting-started-guides/windows/index.md]: @alinbalutoiu can we add support for hyper-v isolation in your Windows Services work as well?

**Pull request merged**